src/pg_to_evalscript/javascript_processes/order.js

```diff
@@ -0,0 +1,63 @@
+function order(arguments) {
+  const { data, asc = true, nodata = null} = arguments;
+
+  validateParameter({
+    processName: "order",
+    parameterName: "data",
+    value: data,
+    required: true,
+    array: true,
+  });
+
+  validateParameter({
+    processName: "order",
+    parameterName: "asc",
+    value: asc,
+    required: true,
+    nullable: false,
+    allowedTypes: ["boolean"],
+  });
+
+  validateParameter({
+    processName: "nodata",
+    parameterName: "nodata",
+    value: nodata,
+    required: true,
+    allowedTypes: ["boolean"],
+  });
+
+  const dataWithIndex = [];
+  for (let i = 0; i < data.length; i++) {
```

Comment:
Checking each value in the array with the `validateParameter` function would also be quite nice IMO

Reply:
We are already checking each value and throwing an error if there is no match.

While it does not produce the same error type I find it is more (future) error proof as value checking is done in one place instead of duplicating the condition logic.

Reply:
Ok yes you have a point, I agree with you. Resolving